schema_based_on_diagram.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE patients (
+    id serial  PRIMARY KEY,
+    name VARCHAR(100),
+    date_of_birth date
+);
+
+CREATE TABLE invoices (
+    id serial  PRIMARY KEY,
+    total_amount decimal,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id int,
+    FOREIGN KEY ( medical_history_id) REFERENCES  medical_histories(id),
+);
+
+CREATE TABLE medical_histories (
+    id serial  PRIMARY KEY,
+    admitted_at  timestamp,
+    patient_id int,
+    status varchar(100)
+     FOREIGN KEY ( patient_id) REFERENCES  patients (id),
+);
+
+CREATE TABLE treatments (
+    id serial  PRIMARY KEY,
+    type VARCHAR(100),
+    name VARCHAR(100)
+);
+
+CREATE TABLE invoice_items (
+    id serial  PRIMARY KEY,
+    unit_price  decimal,
+    quantity int,
+    total_price  decimal,
+    invoice_id  int, 
+    treatment_id int,
+    FOREIGN KEY ( invoice_id) REFERENCES  invoices (id),
+    FOREIGN KEY ( treatment_id) REFERENCES treatments (id),
+);
+
+CREATE INDEX idx_treatment_id ON medical_histories_treatments (treatment_id);
+CREATE INDEX idx_medical_history_id ON medical_histories_treatments (medical_history_id);
+
```

Comment:
### Adding the review again as you deleted my initial review comment
### Add the missing indexes on ALL THE FOREIGN KEYS


- Hey, :wave: Please, let's not forget also to the indexes on ALL THE FOREIGN KEYS  to meet the project requirement. see an example:

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
```
This should be added for ALL THE FOREIGN KEYS, that's just a few from the rest.